packages/channels/botpress-channel-microsoft/CHANGELOG.md

```diff
@@ -20,31 +20,31 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 **Note:** Version bump only for package @botpress/channel-microsoft
 
 <a name="10.24.4"></a>
-## [10.24.4](https://github.com/botpress/botpress/compare/v10.24.3...v10.24.4) (2018-07-20)
+## 10.24.4 (2018-07-20)
 
 
 
 
 **Note:** Version bump only for package @botpress/channel-microsoft
 
 <a name="10.24.3"></a>
-## [10.24.3](https://github.com/botpress/botpress/compare/v10.24.1...v10.24.3) (2018-07-20)
+## 10.24.3
 
 
 
 
 **Note:** Version bump only for package @botpress/channel-microsoft
 
 <a name="10.24.2"></a>
-## [10.24.2](https://github.com/botpress/botpress/compare/v10.24.1...v10.24.2) (2018-07-20)
```

Comment:
Why do we drop the links in docs? Not that I'm specifically pro or against them, just asking

Reply:
We deprecated this version.